ChakraCore Samples/CrossPlatform/README.md

```diff
@@ -0,0 +1,28 @@
+### Embedding ChakraCore on Linux and macOS
```

Comment:
+1 on having this README update separately. 

Also, prepared https://obastemur.com/2016/07/18/how-to-compile-chakracore-on-macososx/ for basic instructions on compiling ChakraCore on OSX. Feel free to take the steps from there and add into our Wiki. Our current OSX requirement is 10.10+ (it is possible that 10.8+ is also fine but I haven't tested it there.

Please mention `PLATFORM=darwin` option for `make` command (similar to my README) in case the OS is macOS.